modules/test/playwright/tests/portlet-configuration-css-web/textStyle.spec.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {pageViewModePagesTest} from '../../fixtures/pageViewModePagesTest';
+import {pagesAdminPagesTest} from '../../fixtures/pagesAdminPagesTest';
+import {productMenuPageTest} from '../../fixtures/productMenuPageTest';
+import {uiElementsPageTest} from '../../fixtures/uiElementsTest';
+import getRandomString from '../../utils/getRandomString';
+
+export const test = mergeTests(
+	loginTest(),
+	pagesAdminPagesTest,
+	pageViewModePagesTest,
+	productMenuPageTest,
+	uiElementsPageTest
+);
+
+test('LPD-31641 - Look & Feel, center text alignment test for LPD-16327', async ({
+	page,
+	pagesAdminPage,
+	productMenuPage,
+	uiElementsPage,
+	widgetPagePage,
+}) => {
+	await productMenuPage.openProductMenuIfClosed();
+	await productMenuPage.goToPages();
+	await uiElementsPage.clickNewButton();
+	if (!pagesAdminPage.addPageButton.isVisible) {
+		await uiElementsPage.clickNewButton();
+	}
+	await pagesAdminPage.addPageButton.waitFor({state: 'visible'});
+	await pagesAdminPage.addPageButton.click();
+
+	const title = getRandomString();
+	await pagesAdminPage.addWidgetPage(title);
+	await productMenuPage.clickSpecificPage(title);
+	await widgetPagePage.addPortlet('Asset Publisher');
+
+	await widgetPagePage.openLookAndFeel();
+	await page
+		.frameLocator('iframe[title="Look and Feel Configuration"]')
+		.getByRole('tab', {name: 'Text Styles'})
+		.click();
+	await page
+		.frameLocator('iframe[title="Look and Feel Configuration"]')
+		.getByLabel('Alignment')
+		.selectOption('center');
+	await page
+		.frameLocator('iframe[title="Look and Feel Configuration"]')
+		.getByRole('button', {name: 'Save'})
+		.click();
+	await expect(
+		page
+			.frameLocator('iframe[title="Look and Feel Configuration"]')
+			.getByLabel('Alignment')
+	).toHaveValue('center');
+
+	await page.goto('/');
```

Comment:
if you use the `{isolatedSiteTest}` fixture, all this is created in a new site and deleted automatically when test finished, so you don't need to delete all this manually. Please give a try.

Reply:
looks better with this isolatedSiteTest :) thank you!